kernel/build.gradle

```diff
@@ -57,7 +57,7 @@ subprojects {
     ext.executeStaticContent = true
     doLast {
       if (executeStaticContent) {
-        def kernelJson = file('kernel.json').text.replace('__MAIN_CLASS__', mainClassName)
+        def kernelJson = file('kernel.json').text.replace('$MAIN_CLASS', '"' + mainClassName + '"')
```

Comment:
why do we have a variable for main class at all?
since we have a kernel.json file for each kernel and we know the class for each kernel, this could be just a constant right?

Reply:
done.